scipy/optimize/_linprog_highs.py

```diff
@@ -399,7 +399,8 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
         'primal_feasibility_tolerance': primal_feasibility_tolerance,
         'simplex_crash_strategy': simplex_crash_strategy,
         'simplex_dual_edge_weight_strategy': simplex_dual_edge_weight_strategy,
-        'simplex_primal_edge_weight_strategy': simplex_primal_edge_weight_strategy,
+        'simplex_primal_edge_weight_strategy': \
+            simplex_primal_edge_weight_strategy,
```

Comment:
PEP8 now complains `continuation line unaligned for hanging indent`, but aligning it to the right of the colon makes the line too long. Any way to fix this other than changing variable or option names?

Reply:
Because this is inside the brackets of the dictionary, you don't need the backslash for the line continuation.  I just checked this snippet with pycodestyle, and it did not complain:

```
a = {
    'foo': 10,
    'bar':
        20,
    'abc': 99,
}
```

Reply:
Thanks.